### ISSUE TITLE
Mobile - Approval for VPN connection dialog is not coming up when lantern is turned on for the very first time

### ISSUE THREAD

**Issue by sunshine19090** [getlantern/lantern#4829]
Steps:

1. Uninstall and install lantern.
2. Open lantern and turn it on. It is turning ON immediately without the VPN connection approval. 

Also, I am not seeing the key sign that indicates lantern is ON.

**Comment by atavism** [comment]
Even if you uninstalled the app, you previously authorized it to open a VPN connection. The only way to revoke that permission is from the notifications panel.

**Comment by sunshine19090** [comment]
@atavism - I don't see the notification panel too

**Comment by sunshine19090** [comment]
reopening only to discuss.

**Comment by myleshorton** [comment]
The notification panel is in Android itself @sunshine19090 -- i.e. you have to revoke the permission you gave Lantern previously from within the OS.

**Comment by sunshine19090** [comment]
@myleshorton @atavism You are talking about the notification that appears when the screen is locked, right? I don't see that notification when lantern is turned ON. That's where I saw it last time when I tested.

**Comment by myleshorton** [comment]
There are two things here @sunshine19090:

1) The VPN connection dialog you opened the ticket about
2) The ability to revoke that permission from Android

Basically the point is that once you've given Lantern permission one time that VPN connection dialog is not going to appear again unless you revoke it from within Android.

**Comment by sunshine19090** [comment]
Yes @myleshorton, I understood the part that the VPN connection dialog is not going to come up again.  
Now, my question is - where I can see this notification in order to revoke the permission. @atavism told me that it can be revoked from the notification panel but I don't see this lantern notification in notification panel.  I have provided two screenshots of the places where I searched for this notification and I couldn't find the way to revoke. The screenshots are taken after turning ON lantern. 

![screenshot_2016-06-28-16-14-31](https://cloud.githubusercontent.com/assets/11394369/16435694/aefe5b08-3d4c-11e6-89b5-5ddf897b5a73.png)
![screenshot_2016-06-28-16-14-18](https://cloud.githubusercontent.com/assets/11394369/16435695/af016ec4-3d4c-11e6-8146-b6e6ec8cfc75.png)
